examples/v2/step_by_step_guide/step4_use_references/two-vms.yaml

```diff
@@ -26,6 +26,9 @@ resources:
       initializeParams:
         sourceImage: https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/family/debian-9
     networkInterfaces:
+    # The resource's "network value" has been replaced with a
```

Comment:
these comments look like annotations for a reviewer, but shouldn't be in the final committed files